src/ol/ol.js

```diff
@@ -29,6 +29,13 @@ ol.DEFAULT_MIN_ZOOM = 0;
 
 
 /**
+ * @define {number} Default maximum allowed threshold  (in pixels) for
+ *     reprojection triangulation. Default is `0.5`.
+ */
+ol.DEFAULT_RASTER_REPROJ_ERROR_THRESHOLD = 0.5;
```

Comment:
Couldn't hurt to be a bit more verbose and call this `ol.DEFAULT_RASTER_REPROJECTION_ERROR_THRESHOLD` .